### ISSUE TITLE
Tests fail on Node.js master

### ISSUE THREAD

**Issue by isaacs** [tapjs/stack-utils#26]
Using nodejs/node@6d3c85f234fa058da62bf428ce39377291f5d56d

```
> stack-utils@0.4.0 test /Users/isaacs/dev/js/stack-utils
> xo && nyc --reporter lcov --reporter text --cache ava --verbose


/Users/isaacs/dev/js/stack-utils/test/test.js
  276:2  warning  Unexpected "todo" comment  no-warning-comments
  322:2  warning  Unexpected "todo" comment  no-warning-comments
  358:2  warning  Unexpected "todo" comment  no-warning-comments

✖ 3 problems (0 errors, 3 warnings)

  ✖ Forked processes must have an IPC channel
----------|----------|----------|----------|----------|----------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------|----------|----------|----------|----------|----------------|
----------|----------|----------|----------|----------|----------------|
All files |      100 |      100 |      100 |      100 |                |
----------|----------|----------|----------|----------|----------------|

npm ERR! Test failed.  See above for more details.
```

**Comment by isaacs** [comment]
Fails on Node.js 6.x as well.

**Comment by isaacs** [comment]
Fixed on 1.0.0.  Newer ava got past the initial hurdle, but the other issues were a bit stickier.
